ionic_user_interface/src/views/ViewRoute.vue

```diff
@@ -99,8 +104,8 @@
             </ion-card-header>
```

Comment:
Daryl says before the comment cards, add a text to show that comments are below. If not cards are confusing